src/hooks/useAddTokenModal.tsx

```diff
@@ -0,0 +1,283 @@
+import React, { useState, useCallback, useRef, useEffect } from 'react'
```

Comment:
This hook doesn't seem to be used, and maybe it has been replaced by `src/hooks/useBetterAddTokenModal.tsx`

Remove?

Reply:
Oops, yes you are exactly right. I forgot about it. Please remove

Reply:
@Velenir

Reply:
I'll do it.

Reply:
Still used in develop, for example in `TokenOptionItem` but probably YES, although this is not the merge to do it. We should create a branch and merge it to develop